### ISSUE TITLE
Conditional Mixin jar on NeoForge is marked as MINECRAFT_FORGE type.

### ISSUE THREAD

**Issue by SteveKunG** [Fallen-Breath/conditional-mixin#9]
### Info
Minecraft: 1.20.6
Conditional Mixin: 0.6.2
Architect Plugin: 3.4.155
Architectury Loom: 1.6.397
NeoForge: 20.6.119

I just simply add a dependency to my build.gradle on NeoForge.
```
dependencies {
    include(implementation("com.github.Fallen-Breath.conditional-mixin:conditional-mixin-neoforge:0.6.2"))
}
```

### Log
`[22:35:23] [main/WARN] (ModDiscoverer) Found incompatible jar C:\Users\xxxx\.gradle\caches\modules-2\files-2.1\com.github.Fallen-Breath.conditional-mixin\conditional-mixin-neoforge\0.6.2\326c37a8f3eb54878671640a85c10889dc9e36b\conditional-mixin-neoforge-0.6.2.jar with reason MINECRAFT_FORGE. Skipping.`

**Comment by Fallen-Breath** [comment]
Looks like neoforge now hates the `META-INF/mods.toml` file since mc1.20.5, that's a breaking change

See also: https://github.com/neoforged/FancyModLoader/pull/100

**Comment by Fallen-Breath** [comment]
resolved in [v0.6.3](https://github.com/Fallen-Breath/conditional-mixin/releases/tag/v0.6.3)
